client/my-sites/pages/page/index.js

```diff
@@ -670,6 +689,21 @@ class Page extends Component {
 		this.props.recordEvent( 'Clicked Copy Page' );
 	};
 
+	exportPage = () => {
+		this.props.recordEvent( 'Clicked Export Page' );
+		const { page } = this.props;
+
+		const fileContent = JSON.stringify( {
+			__file: 'wp_template',
+			language: 'en',
+			title: page.title,
+			content: page.content,
+		} );
+		const blob = new window.Blob( [ fileContent ], { type: 'application/json' } );
+		const fileName = ( page.title ? page.title : 'page' ) + '.json';
```

Comment:
No big deal, but maybe in a future iteration the file name could include the site from which the page is exported, and a download date stamp in case we want to download multiple versions of the same page?

E.g.,  `sub.domain.com_Page_Name-Tue Mar 24 2020.json`